src/math/src/fast_power.cairo

```diff
@@ -7,10 +7,10 @@
 // * ` modulus ` - The modulus used in the calculation # Returns
 // * ` u128 ` - The result of ( base ^ power ) mod modulus
 
-fn fast_power(base: u128, mut power: u128, modulus: u128) -> u128 {
+pub fn fast_power(base: u128, mut power: u128, modulus: u128) -> u128 {
     // Return invalid input error
     if base == 0 {
-        panic_with_felt252('II')
+        core::panic_with_felt252('II')
```

Comment:
wtf is this error message ?

Reply:
No idea, just updated the code, but should indeed be addressed in a global review of the code
Updated all panic_with to panic macro btw